barretenberg/cpp/src/barretenberg/goblin/goblin.hpp

```diff
@@ -77,68 +69,6 @@ class GoblinProver {
         commitment_key = bn254_commitment_key ? bn254_commitment_key : nullptr;
         GoblinMockCircuits::perform_op_queue_interactions_for_mock_first_circuit(op_queue);
     }
-    /**
-     * @brief Construct a MegaHonk proof and a merge proof for the present circuit.
-     * @details If there is a previous merge proof, recursively verify it.
-     *
-     * @param circuit_builder
-     */
-    GoblinAccumulationOutput accumulate(MegaBuilder& circuit_builder)
```

Comment:
This method is left over from the pre-PG days when we thought we'd simply use vanilla recursion with Goblin as our IVC scheme. More cleanup is needed in Goblin to remove unneeded complexity from those days but I'm getting a start on it here.